### PR TITLE
Added retry request in Hung RequestTiming FAT

### LINKED ISSUE #19849: Test Failure: com.ibm.ws.request.timing.hung.fat.HungRequestTiming.testHungDynamicThresholdUpdate_EE9_FEATURES
RTC 286910

```
testHungDynamicThresholdUpdate_EE9_FEATURES:junit.framework.AssertionFailedError: 2021-09-13-23:06:34:173 No Hung detection warning found!!!
at com.ibm.ws.request.timing.hung.fat.HungRequestTiming.testHungDynamicThresholdUpdate(HungRequestTiming.java:256)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:198)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:319)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:172)
at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:143)
```

### PR DESCRIPTION
**Pull request by pgunapal**
fixes #19849

### CODE REVIEW

**Comment by pgunapal** [comment]
#build

File: dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestTiming.java [hunk 1]
```diff
@@ -263,18 +257,26 @@ public void testHungDynamicThresholdUpdate() throws Exception {
...
-        CommonTasks.writeLogMsg(Level.INFO, "Calling TestWebApp Application with URL=" + url.toString());
-        con = getHttpConnection(url);
-        br = getConnectionStream(con);
-        br.readLine();
+        createRequest(4000);
 
         CommonTasks.writeLogMsg(Level.INFO, "Waiting for hung detection warning");
-        CommonTasks.writeLogMsg(Level.INFO, "----> Hung Request Warning 1 : " + line1);
-        server.waitForStringInLogUsingMark("TRAS0114W", 30000);
+        server.waitForStringInLog("TRAS0114W", 30000);
```
Comment (fmhwong) [inline dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestTiming.java]: Should we set `setMarkToEndOfLog()` before we change server configuration and should this line use `waitForStringInLogUsingMark()`?

Comment (pgunapal) [reply dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestTiming.java]: Updated

**Comment by pgunapal** [comment]
#build

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by pgunapal** [comment]
#libby

**Pull request merged**